sui_programmability/examples/basics/sources/loud_wrapper.move

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/// An example of data + Event wrapping. Allows attaching `Event` to a wrapper
+/// so whenever an Wrapper is destroyed, a user-defined event is emitted
+module basics::loud_wrapper {
```

Comment:
definitely neat! not sure if we need the change to `event` to do this though?

Reply:
Yeah, haha. I removed changes to Event module.

Reply:
But now that I removed it doesn't feel right. We aim to have type wrappers for everything, and if someone took this pattern it would've been nice to have a type wrapper for all of them. So for the sake of this example - yes, it is not necessary to have the Event type, but perhaps if we aimed for people coming up with their solutions for this case, it would be nice to have it.